Automattic/jetpack
# My Jetpack: Introduce `absent_with_plan` status

Closes #29905 

## Proposed changes:

* Introduce new `absent_with_plan` status.
* Change the label from `Add` to `Get` in Product Card, and from `Get` to `Install` if the user has the required plan.
* Change the label from `Add` to `Get` in Product Detail Card, and from `Get` to `Install` if the user has the required plan.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spawn a new JN site.
* Buy the plan for some product.
* Make sure to have a different standalone plugin installed.
* That product should have the card with `Install X` as the label and absent as the layout.
*  If we go to `Interstitial` of this product, it should show `Install` too.
* You should see other cards (if the product has a required plan) labeled as `Get X`.

## Demo

_Without plan

| Product Card | Interstitial |
| ------------- | ---------- |
| ![CleanShot 2023-04-04 at 16 48 04](https://user-images.githubusercontent.com/1663717/229903642-29a5b930-9db5-41c6-acb0-6514ed1e923e.png) | ![CleanShot 2023-04-04 at 16 48 34](https://user-images.githubusercontent.com/1663717/229903693-05047b5c-cd1e-4761-8869-c8591cc8e390.png) |

_With plan_

| Product Card | Interstitial |
| ------------- | ---------- |
| ![CleanShot 2023-04-04 at 16 50 54](https://user-images.githubusercontent.com/1663717/229904381-172fcf5c-f390-4020-b9bf-8ee040747cdf.png) | ![CleanShot 2023-04-04 at 16 51 10](https://user-images.githubusercontent.com/1663717/229904441-2b115fdb-d716-4b19-a954-c014d3f0095f.png) |
